### DISCUSSION TITLE
An bug when recording conversations

### DISCUSSION THREAD

**Discussion by djgraco** [BearWare/TeamTalk5#1073] [General]
Hello,
I would like to report a recording error for a conversation. Used the latest beta teamtalk v5.8.2.5053
Well, when we record, all changes in user volumes or the master volume itself are saved and included in the recording. Previously, the person could not be heard only when audio capture was turned off (ctrl + 4).
Hope the problem can be solved.
regards

**Comment by bear101** [comment]
The change is intentional. Previously volume settings were not reflected in recording. Now they are. Also if you mute someone then the muted user will not be heard in the recording.
